src/qt/guiutil.cpp

```diff
@@ -774,6 +774,18 @@ QString formatBytes(uint64_t bytes)
     return QObject::tr("%1 GB").arg(bytes / 1'000'000'000);
 }
 
+QString formatBps(uint64_t bytes)
+{
+    if (bytes < 10'000)
+        return QObject::tr("%1 bps").arg(bytes);
+    if (bytes < 10'000'000)
+        return QObject::tr("%1 kbps").arg(bytes / 1'000);
+    if (bytes < 10'000'000'000)
+        return QObject::tr("%1 Mbps").arg(bytes / 1'000'000);
```

Comment:
```suggestion
    if (bytes < 10'000)
        //: "Bit per second."
        return QObject::tr("%1 bps").arg(bytes);
    if (bytes < 10'000'000)
        //: "Kilobit per second."
        return QObject::tr("%1 kbps").arg(bytes / 1'000);
    if (bytes < 10'000'000'000)
        //: "Megabit per second."
        return QObject::tr("%1 Mbps").arg(bytes / 1'000'000);
```

Reply:
Oh, just noted that here are bytes not bits. In that case the correct contraction is upper case "B", not "b": Bps, kBps, MBps.

Reply:
@hebasto actually, they are bits, but I ought to change the variable name to reflect that. Will do so, and thanks for the guidance on the commentation.

Reply:
> Oh, just noted that here are bytes not bits. In that case the correct contraction is upper case "B", not "b": Bps, kBps, MBps.

why is the k small but the M is big?

Reply:
https://en.wikipedia.org/wiki/Kilo-
https://en.wikipedia.org/wiki/Mega-

Reply:
ah, ok. will change.